subcell_analysis/compression_analysis.py

```diff
@@ -1,17 +1,19 @@
 #!/usr/bin/env python
-from typing import Tuple
-import numpy as np
+from typing import Any, Tuple
```

Comment:
We could perhaps move this to `bin` or another subfolder like `compression_analysis`? Having a `.py` file in the main folder feels a bit out of place but it's just a personal preference/nit.

Reply:
I think it's ok, we can move it to it's own dir later if it gets crowded.